backend/pkg/abstraction/logger.go

```diff
@@ -1,11 +1,16 @@
 package abstraction
 
+import "time"
+
 // LoggerName is the name of the logger that manages a piece of data
 type LoggerName string
 
 // LoggerRecord is a piece of data stored by a logger
 type LoggerRecord interface {
 	Name() LoggerName
+	GetFrom() string
+	GetTo() string
+	GetTimestamp() time.Time
```

Comment:
Don't add these methods to the general abstraction because maybe in the future we add a new record type that does not have a source and a destination and then it is up to the actual record struct for the sublogger to pass in those values